aiogram/dispatcher/filters/builtin.py

```diff
@@ -66,8 +66,10 @@ def __init__(self, commands: Union[Iterable, str],
                 @dp.message_handler(commands=['myCommand'], commands_ignore_caption=False, content_types=ContentType.ANY)
                 @dp.message_handler(Command(['myCommand'], ignore_caption=False), content_types=[ContentType.TEXT, ContentType.DOCUMENT])
         """
-        if isinstance(commands, str):
+        if isinstance(commands, (str, BotCommand)):
             commands = (commands,)
+        if isinstance(commands, Iterable):
+            commands = [cmd.command if isinstance(cmd, BotCommand) else cmd for cmd in commands]
```

Comment:
What if `command` is not `Iterable`?

before this change if something bad was passed to the command filter the filter anyway will be broken, but if you already add such thing, please add the `else:` condition with raising corresponding error

Reply:
add `isinstance` checks. but anyway iterable can contains ints, or any other. Should I add `isinctance` check for all iterable member? somthing like:
```python
if not all(map(lambda cmd: isinstance(cmd, str), commands)):
            raise ValueError(..)
```